ReText/editor.py

```diff
@@ -105,6 +105,8 @@ def __init__(self, parent):
 		self.infoArea = LineInfoArea(self)
 		self.statistics = (0, 0, 0)
 		self.statsArea = TextInfoArea(self)
+		self.returnBlockPattern = re.compile("^[\\s]*([*>-]|\\d+\\.) ")
+		self.orderedListPattern = re.compile("^([\\s]*)(\\d+)\\. $")
```

Comment:
I would move it to class level (below the `scrollLimitReached = pyqtSignal(QWheelEvent)` line) for even more performance :)

Reply:
Done :)